src/routes/templates/templates.json

```diff
@@ -776,5 +776,19 @@
 		"category": "SvelteKit",
 		"tags": ["blog", "templates"],
 		"stars": 0
+	},
+	{
+		"title": "Svelte-Typescript-Electron-Webpack",
+		"url": "https://github.com/ryan-way-boilerplate/stew",
+		"description": "Template with svelte typescript electron and webpack",
+		"npm": "",
+		"addedOn": "2022-03-21",
+		"category": "Svelte",
+		"tags": [
+			"typescript",
```

Comment:
you need to run prettier. this is causing `lint` to fail

Reply:
updated